### PR TITLE
docs: grammar and style

### PR DESCRIPTION
**Pull request by krishnevsky**
Minor edits for grammar/style.

### CODE REVIEW

**Comment by levonet** [comment]
\cc @sipayRT @j0tunn

File: README.md [hunk 1]
```diff
@@ -6,42 +6,41 @@
...
-  `outline` properties;
+* Include the `box-shadow` and `outline` properties when calculating element position and size
 
-* Some special case differences between images (rendering artifacts, text caret,
-  etc.) are ignored;
+* Ignore some special case differences between images (rendering artifacts, text caret,
+  etc.)
 
-* CSS test coverage statistics.
+* Get CSS test coverage statistics
```
Comment (tormozz48) [inline README.md]: Get -> Gather

Comment (krishnevsky) [inline README.md]: ок

**Comment by sipayRT** [comment]
squash commits pls

**Pull request merged**